app/pages/admin.js

```diff
@@ -91,25 +100,21 @@ function Admin() {
       });
   };
 
+  async function onCheckChange(org) {
+    org.isHidden = !org.isHidden;
+    updateOrg(org);
+  }
+
   const contentWhitelist = (
```

Comment:
This can be done in a subsequent PR, for now I think it's okay to leave in as legacy code in case we want to switch back to the whitelist verification for whatever reason.